scripts/pre-commit

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+echo "*******************************"
+echo   "Running git pre-commit hook"
```

Comment:
Can you add a message saying this will only format Kotlin files, for non kotlin files, please run ./gradlew checkstyle and review the results separately. 

You can also update this script to only work on files that have changed so take in the files that were touched and then run the lint on it.

Reply:
Here is the sequence of steps as I understand them: 
- git add .
- git commit -m "Blah blah blah" <-- this is when the script is run and the file is changed
- git push

If there were files changed by the ktLintFormat they will become unstaged. Is that correct?